opentelemetry-sdk/src/export/logs/mod.rs

```diff
@@ -14,6 +16,11 @@ pub trait LogExporter: Send + Debug {
     async fn export(&mut self, batch: Vec<LogData>) -> LogResult<()>;
     /// Shuts down the expoter.
     fn shutdown(&mut self) {}
+    //#[cfg(feature = "logs_level_enabled")]
+    /// Chek if logs are enabled.
+    fn event_enabled(&self, _level: Severity, _target: &str, _name: &str) -> bool {
```

Comment:
the method in the API does not have the third parameter (name) ? 
`fn event_enabled(&self, level: Severity, target: &str) -> bool;`

Reply:
Yes, the third parameter is part of `LogProcessor::event_enabled` and `LogExporter::event_enabled`. The third parameter is logger-name, which is added by logger.